src/Symfony/Component/DependencyInjection/Tests/Fixtures/Preload/DummyWithInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Tests\Fixtures\Preload;
+
+final class DummyWithInterface implements \NonExistentDummyInterface
+{
+    public A $a;
+
+    public function doSomething(B $b): ?C
+    {
+        return null;
+    }
+
+    public function noTypes($foo)
+    {
+    }
+
+    public function builtinTypes(int $foo): ?string
+    {
+    }
```

Comment:
```suggestion
```

This is just noise. What we want to assert is that the preloader does not choke on the missing `NonExistentDummyInterface`, so we don't need the rest of the class body.